config/tests/samples/create/contents_test.go

```diff
@@ -15,11 +12,12 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
-var matchEverythingRegex = regexp.MustCompile(".*")
```

Comment:
Maybe it's still reasonable to keep a global variable representing this regex? But maybe in samples.go instead?

Reply:
My 2c is that we should add it when/if we need it.  I don't think it will be very hard to recreate if we do need it!